ml-agents/mlagents/trainers/tf_policy.py

```diff
@@ -113,18 +114,22 @@ def _load_graph(self):
                 )
             self.saver.restore(self.sess, ckpt.model_checkpoint_path)
 
-    def evaluate(self, batched_step_result: BatchedStepResult) -> Dict[str, Any]:
+    def evaluate(
+        self, batched_step_result: BatchedStepResult, global_agent_ids: List[str]
+    ) -> Dict[str, Any]:
         """
         Evaluates policy for the agent experiences provided.
-        :param brain_info: BatchedStepResult input to network.
+        :param batched_step_result: BatchedStepResult input to network.
         :return: Output from policy based on self.inference_dict.
         """
         raise UnityPolicyException("The evaluate function was not implemented.")
 
-    def get_action(self, batched_step_result: BatchedStepResult) -> ActionInfo:
+    def get_action(
```

Comment:
docstring for worker_id. Also, maybe this comment should live on the abstract method in policy.py ?

Reply:
Added docstring